src/background.js

```diff
@@ -17,6 +17,8 @@ const ignoredPages = new Set([
   'wallet',
 ]);
 
+let lastChatSize = {};
```

Comment:
Please provide a description and type (`/** @type {foo} */`) for this global.

Reply:
Done, I also renamed it to be more adequate to what it stores.